Version 3/3.3.0/HUSST_Ergebnisdaten_3_3_0.xsd

```diff
@@ -1684,6 +1685,26 @@
       <element name="Zeitpunkt" type="dateTime" maxOccurs="1" minOccurs="0"/>
     </sequence>
   </complexType>
+  
+  <complexType name="WebShopBezahlung_Type">
+    <sequence>
+      <element name="TerminalId" type="string" maxOccurs="1" minOccurs="1">
+		<annotation>
+          <documentation>Eindeutige Terminal-ID bei Verwendung eines gesonderten Kartenzahlungsterminals.</documentation>
+        </annotation>
+	  </element>
+      <element name="TransaktionsNr" type="string" maxOccurs="1" minOccurs="1">
+		<annotation>
+          <documentation>Vom Zahlungsdienstleister vergebene, eindeutige Transaktionsnummer der durchgeführten Zahlungstransaktion.</documentation>
+        </annotation>
+	  </element>
+      <element name="Zeitpunkt" type="dateTime" maxOccurs="1" minOccurs="1">
+		<annotation>
+          <documentation>Zeitpunkt, zu dem die Zahlungstransaktion beim Zahlungsdienstleister registriert wurde.</documentation>
+        </annotation>
+	  </element>
+    </sequence>
+  </complexType>
```

Comment:
Hätten wir hier nicht auch die ZVT-Zahlung verwenden können. Der Name passt dann nicht ganz, aber es sind doch in erster Linie alles unbare bzw. elektronische Zahlungsmittel.